deploy/eventlistener.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: triggers.tekton.dev/v1beta1
+kind: EventListener
+metadata:
+  name: kuadrant-nightly-update-listener
+spec:
+  serviceAccountName: pipeline
+  triggers:
+    - name: kuadrant-nightly-update-trigger
+      bindings:
+        - name: istio-provider
+          value: ossm3
+        - name: kube-api
+          value: https://kubernetes.default
```

Comment:
The login task has problem with this url - the sed command, can it be fixed in future PR? @averevki

Reply:
Create new task if kubernetes.default is what is supposed to be used in the deploy pipeline. Existing login task accepts only named URLs and is able to work with multiple clusters in a solo pipeline